### ISSUE TITLE
Can't import on Python3.8 (because of cloudpickle==0.4)

### ISSUE THREAD

**Issue by carderne** [descarteslabs/descarteslabs-python#233]
## Expected Behavior
Able to import and use `descarteslabs==1.3.0` with `Python 3.8.2`.

## Actual Behavior
```
import descarteslabs as dl
```
```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-2-316e5ca2d1c8> in <module>
----> 1 import cloudpickle

~/.envs/env/lib/python3.8/site-packages/cloudpickle/__init__.py in <module>
      1 from __future__ import absolute_import
      2 
----> 3 from cloudpickle.cloudpickle import *
      4 
      5 __version__ = '0.4.0'

~/.envs/env/lib/python3.8/site-packages/cloudpickle/cloudpickle.py in <module>
    143 
    144 
--> 145 _cell_set_template_code = _make_cell_set_template_code()
    146 
    147 

~/.envs/env/lib/python3.8/site-packages/cloudpickle/cloudpickle.py in _make_cell_set_template_code()
    124         )
    125     else:
--> 126         return types.CodeType(
    127             co.co_argcount,
    128             co.co_kwonlyargcount,

TypeError: an integer is required (got type bytes)
```

## Steps to Reproduce
  1. `pip install descarteslabs==1.3.0`
  2. Try to import `descarteslabs`

## Workarounds
Have force-installed `cloudpickle==1.5.0`, as it is pinned to `0.4.0` in descarteslabs' `setup.py`.

## Environment
- Ubuntu 20.04
- Python 3.8.2

**Comment by stephaniegott** [comment]
Thanks for reporting this! It is a known issue and one we are looking at resolving it on our end (unfortunately there is more involved than simply pinning a newer version of `cloudpickle`). The bottom line, however, is that `descarteslabs` does not support Python 3.8 yet. The versions we do support are listed on our [PyPI page](https://pypi.org/project/descarteslabs/).

**Comment by stephencpope** [comment]
@carderne we have just released client version 1.5.0 which includes support for Python 3.8. Please see https://github.com/descarteslabs/descarteslabs-python/tree/v1.5.0#changelog for more info!
